src/Migration/Sources/Appwrite.php

```
@@ -1996,6 +2035,99 @@ private function exportSiteDeploymentData(Site $site, array $deployment): void
+                $report[Resource::TYPE_PLATFORM] = $response['total'] ?? 0;
+            } catch (\Throwable) {
+                $report[Resource::TYPE_PLATFORM] = 0;
+            }
+        }
+    }
+
+    /**
+     * @param int $batchSize
+     * @param array<string> $resources
+     */
+    protected function exportGroupIntegrations(int $batchSize, array $resources): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$batchSize'. (undefined)

(UnusedFormalParameter)